cmd/kubernetes_version.go

```diff
@@ -122,21 +148,9 @@ func loadCertficate(certFile string) (*tls.Certificate, error) {
 	return &tlsCert, nil
 }
 
-func getKubernetesURL() string {
-	k8sVersionURL := "https://kubernetes.default.svc/version"
-
-	// The following provides flexibility to use
-	// K8S provided variables is situations where
-	// hostNetwork: true
-	if !isEmpty(os.Getenv("KUBE_BENCH_K8S_ENV")) {
-		k8sHost := os.Getenv("KUBERNETES_SERVICE_HOST")
-		k8sPort := os.Getenv("KUBERNETES_SERVICE_PORT_HTTPS")
-		if !isEmpty(k8sHost) && !isEmpty(k8sPort) {
-			return fmt.Sprintf("https://%s:%s/version", k8sHost, k8sPort)
-		}
-
-		glog.V(2).Info(fmt.Sprintf("KUBE_BENCH_K8S_ENV is set, but environment variables KUBERNETES_SERVICE_HOST or KUBERNETES_SERVICE_PORT_HTTPS are not set"))
+func getKubernetesURLs() []string {
+	return []string{
+		"https://kubernetes.default.svc/version",
+		fmt.Sprintf("https://%s:%s/version", os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT_HTTPS")),
```

Comment:
This doesn't use the environment variables if they were set, unless `kubernetes.default.svc` fails. If the env vars are set, that's what kube-bench should use, so I don't think there's any reason to change this function or to have it return a list. 

Also this is no longer checking `KUBE_BENCH_K8S_ENV` and the tests below seem to remove it. If you are retiring the use of some env vars I think you should document that in the comments about the PR so that whoever is reviewing it understands why you're doing it.